workers/unity/Packages/com.improbable.gdk.core/Systems/CleanReactiveComponentsSystem.cs

```diff
@@ -87,6 +87,18 @@ private void AddRemoveComponentAction<T>()
                 }
             });
         }
+        
+        private void UsedOnlyForAotCodeGeneration()
+        {
+            // IL2CPP needs those lines for AOT instantiation of methods
+            AddRemoveComponentAction<OnConnected>();
+            AddRemoveComponentAction<OnDisconnected>();
+            AddRemoveComponentAction<NewlyAddedSpatialOSEntity>();
```

Comment:
It doesn't look like there's generic way to declare such methods for AOT.
Not sure if there's any other way to address this issue (some kind of code generation may be).

Reply:
that is unfortunate... maybe we need to refactor this system or, as you said, find a way to codegenerate it

Reply:
one idea jamie and I had is to hook into Unity's build system. Before actually running the build, use reflection to find all tags that have the `RemoveAtEndOfTick`attribute and generate this method. not sure how nice that will be, but would probably be the best solution unless we do a redesign.

relevant docs:
https://docs.unity3d.com/Manual/BuildPlayerPipeline.html
https://docs.unity3d.com/ScriptReference/Build.IPreprocessBuildWithReport.html
https://docs.unity3d.com/ScriptReference/Build.IPreprocessBuildWithReport.OnPreprocessBuild.html

Reply:
That is assuming that you can generate files that are then included in the builds step